internal/client/client.go

```diff
@@ -105,7 +106,14 @@ func (client *TroccoClient) do(
 		if err != nil {
 			return fmt.Errorf("%s", resp.Status)
 		}
-		return fmt.Errorf("%s", errorOutput.Error)
+
+		if errorOutput.Error != "" {
+			return fmt.Errorf("%s", errorOutput.Error)
+		} else if errorOutput.Message != "" {
+			return fmt.Errorf("%s", errorOutput.Message)
+		} else {
+			return fmt.Errorf("status code is %s", resp.Status)
```

Comment:
Just to be safe, if neither error nor message has a value, only statusCode is returned.